### PR TITLE
Updated from forks and my things (0.5.20)

### PR DESCRIPTION
**Pull request by rickerd**
Changed npm-> yarn commands

- Linted everything in src/ .js,.jsx
- Placed header and footer outside the container to let more css features be possible.
- Added make scroll optional
- Updated README
- Added optional loading spinner

With this MR all features are added just fine. And other features are welcome of course. All you need to do is make a new version because i already built the new version into dist/

Know it better? Need changes? Do it yourself, i ain't developing for this repo anymore.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -10,7 +10,7 @@ A simple, responsive lightbox component for displaying an array of images.
 
 
 ```bash
-npm install --save react-images
+yarn add react-images --save
```
Comment (aslafy-z) [inline README.md]: `--save` switch does not exists with yarn. It adds it automatically to dependencies.

File: src/Lightbox.js [hunk 2]
```diff
@@ -252,7 +267,7 @@ class Lightbox extends Component {
 
 		const thumbnailsSize = showThumbnails ? this.theme.thumbnail.size : 0;
 		const heightOffset = `${this.theme.header.height + this.theme.footer.height + thumbnailsSize
-			+ (this.theme.container.gutter.vertical)}px`;
+		+ (this.theme.container.gutter.vertical)}px`;
```
Comment (aslafy-z) [inline src/Lightbox.js]: Is lint happy with this ?

File: .github/CONTRIBUTING.md [hunk 3]
```diff
@@ -8,12 +8,12 @@ Before you open a PR:
...
 especially if you're going to spend a lot of time on it.
-* In development, run `npm start` to build (+watch) the project source, and run
+* In development, run `yarn start` to build (+watch) the project source, and run
 the [development server](http://localhost:8000).
 * Please ensure all the examples work correctly after your change. If you're
 adding a major new use-case, add a new example demonstrating its use.
 * Please **DO NOT** commit the build files. Make sure **ONLY** your changes to
 `/src/` and `/examples/src` are included in your PR.
-* Be careful to follow the code style of the project. Run `npm run lint` after
+* Be careful to follow the code style of the project. Run `yarn run lint` after
```
Comment (aslafy-z) [inline .github/CONTRIBUTING.md]: I would push yarn cli-style here by hinting `yarn lint` instead

File: README.md [hunk 4]
```diff
@@ -41,8 +41,8 @@ Live demo: [jossmac.github.io/react-images](http://jossmac.github.io/react-image
 To build the examples locally, run:
 
 ```
-npm install
-npm start
+yarn install
```
Comment (aslafy-z) [inline README.md]: I would use `yarn` only here

File: src/Lightbox.js (hunk 2 above)
Comment (rickerd) [reply src/Lightbox.js]: Yes lint is happy. If you doubt me, checkout my repo and lint it yourself.

File: .github/CONTRIBUTING.md (hunk 3 above)
Comment (rickerd) [reply .github/CONTRIBUTING.md]: changed

File: README.md (hunk 1 above)
Comment (rickerd) [reply README.md]: changed

File: README.md (hunk 4 above)
Comment (rickerd) [reply README.md]: changed

**Comment by rickerd** [comment]
Since @aslafy-z knows best but doesn't change a sh*t himself i've changed it for him so he is pleased with my code.
Now we'll wait another 2 years before everything is merged since another thousand of PR are sitting ducks.

I've forked it and changed it for my needs, i don't care about the PR anymore do whatever you want with it.

**Comment by jorrit** [comment]
You should restrain yourself in order to come across as a sane person.

Are you publishing your fork on NPM?

**Comment by rickerd** [comment]
I really don't give a damn.
I have my thing working how i want it to work.
The repo hasn't been touched in a proper way for . along time. I've corrected thing and something is wining about it.. soo.. i don't care anymore if it's merged or not.

Sane or insane, don't care. Closing this sh*t since ... i don't care anymore about this repo or PR.

**Pull request closed without merging**